components/02-elements/cookie-message/CookieMessage.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div
+    :class="['cookie-message', openClass]"
+    :data-type="dataType"
+    :aria-label="ariaLabel"
+    tabindex="0"
+  >
+    <div class="cookie-message__container">
+      <span
+        class="cookie-message__text"
+        v-html="text"
+      />
+      <button
+        class="cookie-message__close"
+        type="button"
+        :aria-label="closeAriaLabel"
+        @click="closeBar"
+      >
+        {{ closeLabel }}
+        <icon
+          v-if="closeIcon"
+          :icon="closeIcon"
+          class="cookie-message__close-icon"
+        />
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+import Icon from '../../01-globals/icon/Icon.vue'
+
+export default {
+  components: { Icon },
+  props: {
+    ariaLabel: {
+      type: String,
+      required: true
+    },
+    text: {
+      type: String,
+      required: true
+    },
+    dataType: {
+      type: String,
+      required: true
+    },
+    closeIcon: {
+      type: [Boolean, String],
+      required: false,
+      default: false
+    },
+    closeLabel: {
+      type: String,
+      required: false,
+      default: ''
+    },
+    closeAriaLabel: {
+      type: String,
+      required: false,
+      default: ''
+    }
+  },
+  data: () => {
+    return {
+      openClass: ''
```

Comment:
Maybe more readable would be to have `isOpen: false`, and setting the value to true if it is open. 
In line 3: `:class="['cookie-message', isOpen && 'cookie-message--open']"` - then all classes stay in template. What do you think?